lib/pages/usersProfileDetails.dart

```diff
@@ -10,11 +12,11 @@ class UsersProfileDetails extends StatefulWidget {
   UsersProfileDetails({this.details});
 
   @override
-  _UsersProfileDetailsPageState createState() =>
-      _UsersProfileDetailsPageState();
+  UsersProfileDetailsPageState createState() =>
```

Comment:
This is wrong. You never make a inner class which extends State<T> public. It should be private.
Hence the "_" like in _UserProfileDetailsPageState. If you want to use the functions inside this class you have to call it from outside. outer_class.inner_class.method(). For Questions: contanct me